src/components/cities/modals/CityModal.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  Button,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Divider,
+  Modal,
+} from '@mui/material'
+import { Dialog } from '@mui/material'
+import { Box } from '@mui/material'
+
+export default function CityModal(props: any) {
```

Comment:
Please avoid using `any` as a type

Nobody will really know what the expected `props` for `CityModal` are